services/bettersocial/chat/allAnonChat.js

```diff
@@ -7,7 +7,8 @@ const generate_channel_id_for_anon_chat = (owner, member, context = null) => {
   const hash = crypto.createHash('sha256');
   hash.update(owner);
   hash.update(member);
-  hash.update(context);
+  if (context) hash.update(context);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made to check if `context` exists before updating the hash is a good one. It prevents potential errors when `context` is undefined or null.
```diff
-   hash.update(context);
+   if (context) hash.update(context);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->